### PR TITLE
chore(docs): convert to dochub

### LINKED ISSUE #239: Move docs to hub
Move docs and code samples to hub.  During transition leave the docs in the README.md until fully merged.

### PR DESCRIPTION
**Pull request by PWKad**
Clean up readme and move to dochub docs
Fixes #239

### CODE REVIEW

File: doc/article/en-US/dialog-basics.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+If you use TypeScript, install the plugin's typings with the following command
+
+```shell
+typings install github:aurelia/dialog --save
+  ```
+
+2. Make sure you use [manual bootstrapping](http://aurelia.io/docs#startup-and-configuration). In order to do so open your `index.html` and locate the element with the attribute aurelia-app. Change it to look like this:
+
+<code-listing heading="index.html">
+  <source-code lang="ES 2015">
```
Comment (EisenbergEffect) [inline doc/article/en-US/dialog-basics.md]: lang should be HTML

File: doc/article/en-US/dialog-basics.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+      this.person = person;
+    }
+  }
+  </source-code>
+</code-listing>
+
+  and the corresponding view -
+
+<code-listing heading="edit-person.html">
+  <source-code lang="ES 2015">
```
Comment (EisenbergEffect) [inline doc/article/en-US/dialog-basics.md]: lang should be HTML

File: doc/article/en-US/dialog-basics.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+  </template>
+  </source-code>
+</code-listing>
+
+## [Attach Focus custom attribute](aurelia-doc://section/4/version/1.0.0)
+
+The library exposes an `attach-focus` custom attribute that allows focusing in on an element in the modal when it is loaded.  You can use this to focus a button, input, etc...  Example usage -
+
+<code-listing heading="edit-person.html">
+  <source-code lang="ES 2015">
```
Comment (EisenbergEffect) [inline doc/article/en-US/dialog-basics.md]: lang should be HTML

File: doc/article/en-US/dialog-basics.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+      </ai-dialog-body>
+    </ai-dialog>
+  </template>
+  </source-code>
+</code-listing>
+
+You can also bind the value of the attach-focus attribute if you want to alter which element will be focused based on a view model property.
+
+<code-listing heading="edit-person.html">
+  <source-code lang="ES 2015">
```
Comment (EisenbergEffect) [inline doc/article/en-US/dialog-basics.md]: lang should be HTML

File: doc/article/en-US/dialog-basics.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+</code-listing>
+
+## [Styling the Dialog](aurelia-doc://section/7/version/1.0.0)
+
+## Overlay with 50% opacity
+
+Bootstrap adds 50% opacity and a background color of black to the modal.  To achieve this in dialog you can simply add the following CSS -
+
+<code-listing heading="welcome.js">
+  <source-code lang="ES 2015">
```
Comment (EisenbergEffect) [inline doc/article/en-US/dialog-basics.md]: lang should be CSS

**Comment by PWKad** [comment]
@EisenbergEffect fixed the lang I'll wait to merge this until right before next releases so that the docs don't go dark in between that time.  I think we want to get the TS conversion in as well so once that is in place we'll kick off the release process.

**Pull request closed without merging**